docs/tracking.md

```diff
@@ -63,7 +65,7 @@ please open an issue on [this issue tracker](https://github.com/Quansight-Labs/f
 | meson-python         |   [:simple-github:](https://github.com/mesonbuild/meson-python/issues/499)   | :material-check-bold: |   :material-check-bold:    |           0.16.0           |                                                            |                                                                                              |
 | ml-dtypes            |      [:simple-github:](https://github.com/jax-ml/ml_dtypes/issues/228)       | :material-check-bold: |   :material-check-bold:    |           0.5.1            |                                                            |                                                                                              |
 | mlir-python          |    [:simple-github:](https://github.com/llvm/llvm-project/issues/105522)     | :material-check-bold: |                            |           20.1.0           |                                                            |                                                                                              |
-| multidict            |     [:simple-github:](https://github.com/aio-libs/multidict/issues/1014)     | :material-check-bold: |                            |                            |                                                            |                                                                                              |
+| multidict            |     [:simple-github:](https://github.com/aio-libs/multidict/issues/1014)     | :material-check-bold: |                            |           6.2.0            |                                                            |                                                                                              |
```

Comment:
This should have a check for Release on PyPI as well

Reply:
Good catch. Thanks!